OpenRA.Game/Game.cs

```diff
@@ -457,6 +465,28 @@ static string ChooseShellmap()
 			return shellmaps.Random(CosmeticRandom);
 		}
 
+		public static void SwitchToExternalMod(ExternalMod mod, string[] launchArguments = null, Action onFailed = null)
+		{
+			try
+			{
+				var argsString = mod.LaunchArgs.Append(launchArguments)
+					.Select(a => "\"" + a + "\"").JoinWith(" ");
+
+				var p = Process.Start(mod.LaunchPath, argsString);
```

Comment:
I still don't understand how you are massively misinterpreting the motivations and goals of #12554.  Your points here (and many of your earlier arguments) are problems that we have *right now* that this rework is explicitly going to fix.

OpenRA already reinvented software distribution by having a shared engine and mod launcher that all mods are expected to integrate with.  We have demonstrated quite clearly that this approach has failed for us and for modders, and the points you raise about security show how this fails for users.  Don't forget that mod packages can contain arbitrary executable code, and that we have no mechanisms to verify it through code signing or reproducible builds.

Again, one of the main goals of #12554 is to remove this DIY software distribution and to require individual mods to use standard OS packaging (which we will provide script templates for).  On all platforms this packaging includes support for signed installers, and where available can include source packages and reproducible builds.  Most importantly of all, it becomes clear to users that they are installing new software (not just a sandboxed software addon), and that they must use common sense before granting root access to install the software.